createdisk.sh

```diff
@@ -72,12 +72,11 @@ EOF
     # also in case of microshift the ports like 2222, 443, 80 ..etc need to be manually added
     # and OCP/OKD/podman bundles have it disabled by default.
     ${SSH} core@${VM_IP} -- sudo systemctl disable firewalld
-    ${SSH} core@${VM_IP} -- cat /etc/microshift/config.yaml.default > config.yaml
-    ${YQ} eval --inplace ".dns.baseDomain = \"${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\""  config.yaml
-    ${SCP} config.yaml core@${VM_IP}:/home/core
-    ${SSH} core@${VM_IP} -- 'sudo mv /home/core/config.yaml /etc/microshift/config.yaml'
+    ${YQ} eval --inplace ".dns.baseDomain = \"${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\""  00-microshift-dns.yaml
```

Comment:
I think this will cause a file tracked in git to be modified each time `createdisk.sh` is called? This could be a template file instead, from which we generate a yaml file without modifying it in place.

isn't it late to do this at `createdisk.sh` time, ie once the install is done? Maybe it should be created by the kickstart file?

Reply:
Migrated this to kickstart file and remove the dropin file template.